### ISSUE TITLE
Accessibility issues in book pages

### ISSUE THREAD

**Issue by mfairchild365** [unlcms/unl_four#26]
See http://arts.unl.edu/music/unl-clarinet-studio/studio-information

**Comment by mfairchild365** [comment]
@ericras the next WDN report will be published in a few weeks.  These accessibility issues are causing several sites to fail.  It takes 2 weeks to run every site through webaudit, so the sooner this gets done the better. :)

You said that you have made changes to fix these issues on a few of your sites.  Will you be able to port them over to the default styles soon?
